### PR TITLE
update test PolynomFunction

### PR DESCRIPTION
**Pull request by marcheto**
Add test for method "negate()" from file "PolynomialFunction.java"

### CODE REVIEW

File: src/test/java/org/apache/commons/math4/geometry/euclidean/twod/Vector2DTest.java [hunk 1]
```diff
@@ -37,5 +37,6 @@ public void testCrossProduct() {
 
         Vector2D p5 = new Vector2D(2, 1);
         Assert.assertEquals(-1.0, p5.crossProduct(p1, p2), epsilon);
+
```
Comment (atodorov) [inline src/test/java/org/apache/commons/math4/geometry/euclidean/twod/Vector2DTest.java]: Remove this extra line and amend this commit

File: pom.xml [hunk 2]
```diff
@@ -363,6 +363,12 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+    	<groupId>org.jacoco</groupId>
+    	<artifactId>jacoco-maven-plugin</artifactId>
+    	<version>${commons.jacoco.version}</version>
+    	<scope>test</scope>
+    </dependency>
   </dependencies>
```
Comment (atodorov) [inline pom.xml]: Guys,
I'm not sure if we really need this, someone please comment if this should stay or be removed. Thanks,

**Comment by chtompki** [comment]
We also currently work in the develop branch as opposed to master. I know that may seem a tad backwards, but that's where the work is currently going on.

**Comment by atodorov** [comment]
Rebased in #40. Please close this one.

**Comment by marcheto** [comment]
Close this pull request according to previous comment. This pull request is rebased in #40.

**Pull request closed without merging**